ligo/stablecoin/spec.ligo

```diff
@@ -0,0 +1,263 @@
+(* SPDX-FileCopyrightText: 2020 tqtezos
+   SPDX-License-Identifier: MIT *)
+
+(*
+ * Stablecoin types that are specified in
+ * https://github.com/tqtezos/stablecoin/blob/317b9346b287a4f2c2606c4929237d87008fdad4/docs/specification.md
+ *)
+
+(* ------------------------------------------------------------- *)
+
+(*
+ * Parameter types
+ *)
+
+type token_id is nat
+
+const default_token_id : token_id = abs (0);
```

Comment:
```suggestion
const default_token_id : token_id = 0n;
```

Reply:
Wtf, I clearly remember changing that, wait a minute